packages/testing/config/storybook/main.js

```diff
@@ -131,7 +131,9 @@ const baseConfig = {
 
     return sbConfig
   },
-  staticDirs: [staticAssetsFolder],
+  ...(process.env.NODE_ENV !== 'production' && {
+    staticDirs: [`${staticAssetsFolder}`],
+  }),
```

Comment:
@virtuoushub @jtoar  I'm getting caught up to speed here so that you in advance for the redundancy. What's the case where Node_Env == production? And, when it does, what's the default for staticDirs?

Thanks!

Reply:
@thedavidprice my understanding (local testing supports this) is `build`ing Storybook is when `NODE_ENV` is `'production'` and `start`ing Storybook is when ``NODE_ENV` is something other than `'production'`` (e.g. `development`)

See: https://github.com/redwoodjs/redwood/pull/3515/commits/0067d361d5dc0df213a542f79b8a3a12541e3e5d (notice `static-dir` was only applied when not building)
```sh
!build && `--static-dir "${staticAssetsFolder}"`,
```

See also: https://storybooks.netlify.app/configurations/custom-webpack-config/#full-control-mode

Reply:
Wowza. Great find +catch. Curious what effect this will have but I definitely am +1 to find out.

Should I go ahead and merge as is? Or is there anything else you'd like to change before I do?

Reply:
Safe to merge as is I believe, should result in #3842 passing CI/CD